src/main/asciidoc/reference/neo4j-repositories.adoc

```diff
@@ -482,8 +482,12 @@ Below are some excellent resources:
 
 You can start a read only transaction by marking a class or method with `@Transactional(readOnly=true)`.
 
+[NOTE]
+A custom cypher provided by using the `@Query` annotation will automatically force a read/write transaction.
+This is a known limitation because Neo4j-OGM ignores the `readOnly` flag if custom cypher is provided.
+
 [CAUTION]
-Note that if you open a read only transaction from a service method and then call a mutating method that is marked as read/write your transaction semantics will always be defined by the outermost transaction. 
+Note that if you open a read only transaction from a service method and then call a mutating method that is marked as read/write your transaction semantics will always be defined by the outermost transaction.
```

Comment:
I don't understand this.

I'll get a read transaction, right? Should we link my article about validating transactions? Or the docs? Or the snippet how to do this.

Reply:
Could you please suggest something for this? I think it should then get out of the "caution box". The commit touches this because of the trailing whitespace but there was no intention to change anything.

Reply:
Something like "Be careful when nesting transactions. Spring's `@Transactional` propagates by default outer transactions and doesn't validate whether they are compatible with inner. See [link to Spring docs.]"